gpflow/utilities/bijectors.py

```diff
@@ -5,17 +5,50 @@
 from .. import config
 from .utilities import to_default_float
 
+
 __all__ = ["positive", "triangular"]
 
 
-def positive(lower: Optional[float] = None):
-    lower_value = config.default_positive_minimum()
-    if lower_value is None:
-        return tfp.bijectors.Softplus()
+_POSITIVE_BIJECTOR_MAP = {
+    "exp": tfp.bijectors.Exp,
+    "softplus": tfp.bijectors.Softplus,
+}
+
+
+class Shift(tfp.bijectors.AffineScalar):
+    """Simple subclass so printed name is cleaner."""
+
+    def __init__(self, shift=None, validate_args=False, name='shift'):
+        super().__init__(shift=shift, validate_args=validate_args, name=name)
+
+
+def positive(lower: Optional[float] = None,
+             bijector: Optional[str] = None) -> tfp.bijectors.Bijector:
+    """
+    Returns a positive bijector (a reversible transformation from real to positive numbers).
+
+    :param lower: lower bound override (defaults to config.default_positive_minimum())
+    :param bijector: bijector method override (defaults to config.default_positive_bijector())
+    :returns: a bijector instance
+    """
+    bijector = _get_base_positive_bijector(bijector)
+    if lower is None:
```

Comment:
```suggestion
    default = config.default_positive_minimum()
    lower = default if lower is None else lower
```

Reply:
You need update two lines instead of one, github doesn't allow to do suggestions for multiple lines.

Reply:
I think the current code is cleaner. It also only fetches the default value when necessary rather than on every call. Any strong preference for this suggestion?

Reply:
:) my preference would be to switch to python 3.8 and use walrus operator. As it is not really possible, therefore don't have a strong opinion about it.

Reply:
Yes! Bring on the walrus...one day